Dev/BitbucketServer/stash-pull-requests.2m.rb

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env ruby
+
+require 'duration'
+require 'json'
+require 'rest-client'
+require 'pp'
+
+# rubocop:disable all
+# <bitbar.title>Bitbucket Server (Stash) Pull Requests</bitbar.title>
+# <bitbar.version>v1.0.0</bitbar.version>
+# <bitbar.author>Adam Snodgrass</bitbar.author>
+# <bitbar.author.github>asnodgrass</bitbar.author.github>
+# <bitbar.desc>Displays a list of open pull requests in for all repositories in a specified project on Bitbucket Server (aka Stash) in which the user is a participant. Results are sorted by last update (ascending), and are prefixed with a character to indicate participation type: A for author, R for reviewer, and nothing for other (e.g. added a comment). Each entry is clickable and will take you to the pull request page.</bitbar.desc>
+# <bitbar.image>http://i.imgur.com/9vp4D6e.png</bitbar.image>
+# <bitbar.dependencies>rest-client gem, ruby_duration gem</bitbar.dependencies>
+# rubocop:enable all
```

Comment:
Re-enabled here.  So it's just disabling them for the metadata block I think @keithamus